source/includes/driver-compatibility-caveat.rst

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   The compatibility matrices show which versions of the driver support
+   all of the features of a given major release. Older driver versions
```

Comment:
Unfortunately, I believe this may not be true that it supports "all" of the features... Hence we just write which drivers are recommended for x version.